components/navbar/navbar.css

```diff
@@ -0,0 +1,63 @@
+.main-head {
+  background: var(--complimentary-bg-color);
+  color: var(--secondary-color);
+  /* position: sticky;
+    top: 0px;
+    z-index: 3; */
+}
+
+.nav-component {
+  display: flex;
+  width: 90%;
+  margin: auto;
+  min-height: 10vh;
+  align-items: center;
+  justify-content: space-between;
+  flex-wrap: wrap;
+}
+
+.logo-wrapper {
+  display: flex;
+  align-items: center;
+}
+
+.nav-logo {
+  font-weight: 400;
+}
+
+.hamburger-icon {
+  margin-left: -5rem;
+  cursor: pointer;
+}
+
+.nav-searchBar-container {
+  position: relative;
+}
+
+.nav-searchIcon {
+  position: absolute;
+  top: 10px;
+  left: 12px;
+}
+
+.nav-searchBar {
+  width: 300px;
+  padding-left: 4rem;
+  border: none;
+}
+
+.nav-searchBar:focus {
+  outline: none;
```

Comment:
An element should be focusable, this will make components 
inaccessible to keyboard users.

Reply:
Thanks for pointing that out , I will add a custom focus color to fix that.